### DISCUSSION TITLE
百度网盘操作每次都要点好几下才能生效

### DISCUSSION THREAD

**Discussion by Cyiming** [AlistGo/alist#2249] [General]
比如删除会提示

https://pan.baidu.com/union/doc/

反复操作6-7次后提示成功

**Comment by youtehub** [comment]
![image](https://user-images.githubusercontent.com/18629921/215240453-0414197d-eee2-4905-9376-8fdd252cebd3.png)
是的删除文件时，反复出现这个错误，要点击几次才能正常完成，不知道是不是官方做了限制 ，我上传文件时都要写程序反复重试 
错误内容：Errno: 2, refer to https://pan.baidu.com/union/doc/
文档参考地址：https://pan.baidu.com/union/doc/okumlx17r
![image](https://user-images.githubusercontent.com/18629921/215240477-dc62d0fa-4f4d-4c50-b961-a8491261d313.png)
